setup.py

```diff
@@ -44,6 +44,8 @@ def get_platform():
     return sys.platform
 host_platform = get_platform()
 
+_vxworks = ('vxworks' in host_platform)
```

Comment:
Please rename the variable to VXWORKS to better distinguish local and global variables in functions below.

By the way, I would also prefer to rename host_platform to HOST_PLATFORM for the same reason, but it should be done in a separated PR.

Reply:
Just to confirm: all the variables that are not supposed to be modified should be defined as constants, right? If this is true, besides host_platform, cross_compiling is another one to fix. I looked through PEP-8 but didn't find the guidance.